akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java

```diff
@@ -91,7 +91,7 @@ private void onTrackDevice(DeviceManager.RequestTrackDevice trackMsg) {
     } else {
       log.warning(
           "Ignoring TrackDevice request for {}. This actor is responsible for {}.",
-          groupId,
+              trackMsg.groupId,
```

Comment:
Good observation! I suspect our style checker will complain about the extra indentation here, though...

Reply:
Thanks. What's next step?

Reply:
@erwink Would you be able to run
```
sbt akka-docs/scalafmtAll`
```
and commit the changes and push to your branch again?

Reply:
@patriknw Please send me step by step command I need to run to push eventually push.